core/embed/rust/src/translations/flash.rs

```diff
@@ -64,16 +64,18 @@ pub fn init() {
     }
 }
 
-// SAFETY: Invalidates all references coming from the flash-based blob.
-// In other words, none should exist when this function is called.
+/// # Safety
+/// Invalidates all references coming from the flash-based blob.
+/// In other words, none should exist when this function is called.
```

Comment:
i'm all for doing things the way clippy likes them, but `SAFETY:` is the convention in our codebase and if we want to change it, we should change it everywhere all at once.